packages/components/switch/__tests__/switch.test.tsx

```diff
@@ -299,68 +223,47 @@ describe('Switch.vue', () => {
   })
 
   test('sets checkbox value', async () => {
-    const wrapper = mount({
-      components: {
-        'el-switch': Switch,
-      },
-      template: `
-        <div>
-          <el-switch v-model="value"></el-switch>
-        </div>
-      `,
-      data() {
-        return {
-          value: false,
-        }
-      },
-    })
+    const value = ref(false)
+    const wrapper = mount(() => (
+      <div>
+        <Switch v-model={value.value}></Switch>
```

Comment:
## Suggest

For brevity, `<Switch />` looks good. The same goes for other places.